src/IW/Time.hs

```diff
@@ -0,0 +1,20 @@
+module IW.Time
+       ( firstHaskellRepoCreated
+       , getToday
+       , julianDayToIso
+       ) where
+
+import Data.Time (Day (..), getCurrentTime, utctDay)
+import Data.Time.Format (formatTime, defaultTimeLocale, iso8601DateFormat)
+
+
+-- | This day is equilvalent to 2008-04-03.
```

Comment:
Hmm. this is a very old repo... But I guess it should be fine. Could you try to sync up until this day and tell how much time it requires?

Reply:
@chshersh I will try that rn